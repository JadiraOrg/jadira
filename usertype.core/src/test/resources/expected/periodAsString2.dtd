<!ELEMENT dataset (
    PERIODASSTRING2*)>

<!ELEMENT PERIODASSTRING2 EMPTY>
<!ATTLIST PERIODASSTRING2
    ID CDATA #REQUIRED
    NAME CDATA #IMPLIED
    PERIOD CDATA #IMPLIED
>

