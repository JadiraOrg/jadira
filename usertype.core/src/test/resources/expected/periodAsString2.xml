<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "periodAsString2.dtd">
<!--

  Copyright 2010, 2011, 2012 Christopher Pheby

  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at

      http://www.apache.org/licenses/LICENSE-2.0

  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.

 -->
<dataset>
  <PERIODASSTRING2 ID="0" NAME="test_0" PERIOD="P2D"/>
  <PERIODASSTRING2 ID="1" NAME="test_1" PERIOD="PT30S"/>
  <PERIODASSTRING2 ID="2" NAME="test_2" PERIOD="P3M"/>
  <PERIODASSTRING2 ID="3" NAME="test_3" PERIOD="PT30S"/>
  <PERIODASSTRING2 ID="4" NAME="test_4" PERIOD="P4Y35M40DT141H"/>
  <PERIODASSTRING2 ID="5" NAME="test_5" PERIOD="P28Y10M2DT2H4M35.040S"/>
  <PERIODASSTRING2 ID="6" NAME="test_6" PERIOD="P28Y10MT16H4M35.040S"/>
  <PERIODASSTRING2 ID="7" NAME="test_7" PERIOD="PT16H{Hours}"/>
</dataset>
